salt/modules/boto_vpc.py

```diff
@@ -1759,8 +1759,9 @@ def route_table_exists(route_table_id=None, name=None, route_table_name=None,
                            profile=profile)
 
 
-def route_exists(destination_cidr_block, route_table_name=None, route_table_id=None, gateway_id=None, instance_id=None,
-                 interface_id=None, tags=None, region=None, key=None, keyid=None, profile=None):
+def route_exists(destination_cidr_block, route_table_name=None, route_table_id=None,
+                 gateway_id=None, instance_id=None, interface_id=None, vpc_peering_connection_id=None,
```

Comment:
It looks like these got re-ordered. Could you please restore the original ordering and all the new keyword arguments at the end? Some people depend on calling these positionally.